lib/spack/docs/tutorial_modules.rst

```diff
@@ -678,18 +615,24 @@ Regenerating module files now we obtain:
 
 .. code-block:: console
 
-  root@module-file-tutorial:/# spack module tcl refresh --delete-tree -y
+  $ spack module tcl refresh --delete-tree -y
```

Comment:
The code section below this doesn't seem to be formatting correctly. Looks like plain text rather than a code block.

Reply:
Disregard. Meant other section.